src/create-airseeker.ts

```diff
@@ -0,0 +1,187 @@
+import { chainNameToChainId } from './utils/evm';
+import { Beacons, Gateways, Templates, Triggers } from './utils/airseeker-validation';
+import { sanitiseFilename } from './utils/filesystem';
+
+const questions = (choices: Choice[]): PromptObject[] => {
+  return [
+    {
+      type: 'text',
+      name: 'name',
+      message: ['What is the Airseeker configuration name?'].join('\n'),
+      initial: 'global',
+    },
+    {
+      type: 'autocompleteMultiselect',
+      name: 'selectedBeacons',
+      message: 'What are the beacons you want to include in the configuration?',
+      choices: choices,
+    },
+    //TODO: Add BeaconSet question
+  ];
+};
+
+const main = async () => {
+  const operationsRepository = readOperationsRepository();
+  const documentation = operationsRepository.documentation;
+  const beaconChoices = Object.values(operationsRepository.apis).flatMap((api) =>
+    Object.values(api.beacons).map((beacon) => ({
+      title: `${api.apiMetadata.name}\t${beacon.name}`,
+      value: { api: api, beacon: beacon },
+      selected: true,
+    }))
+  );
+
+  const response = await promptQuestions(questions(beaconChoices));
+  const selectedBeacons = response.selectedBeacons as { api: Api; beacon: Beacon }[];
+  const beacons = selectedBeacons.map((beacon) => beacon.beacon);
+  const apis = selectedBeacons
+    .map((beacon) => beacon.api)
+    .filter((api, index, apis) => apis.findIndex((find) => find.apiMetadata.name === api.apiMetadata.name) === index);
+
+  // Get all the chains the API will be deployed on
+  const apiChains = [...new Set(beacons.flatMap((beacon) => beacon.chains.map((chain) => chain.name)))];
+
+  //// Build airseeker.json ////
+  const airseekerBeacons = beacons.reduce(
+    (beaconObj, beacon) => ({
+      ...beaconObj,
+      [beacon.beaconId]: {
+        airnode: beacon.airnodeAddress,
+        templateId: beacon.templateId,
+        fetchInterval: beacon.airseekerConfig.updateInterval,
+      },
+    }),
+    {} as Beacons
+  );
+
+  const airseekerChains = apiChains
+    .map((chainName) => ({
+      [chainNameToChainId[chainName]]: {
+        contracts: {
+          AirnodeRrp: AirnodeRrpAddresses[chainNameToChainId[chainName]] || '',
+          DapiServer: documentation.chains[chainName].contracts.DapiServer || '',
+        },
+        providers: {
+          provider1: {
+            url: `\${${chainName}_PROVIDER_URL}`.toUpperCase(),
+          },
+        },
+        options: {
+          txType: 'eip1559' as const,
+          priorityFee: {
+            value: 3.12,
+            unit: 'gwei' as const,
+          },
+          baseFeeMultiplier: 2,
+        },
+      },
+    }))
+    .reduce((chainsObject, chain) => ({ ...chainsObject, ...chain }), {});
+
+  const airseekerGateways = apis.reduce(
+    (gatewaysObject, api) => ({
+      ...gatewaysObject,
+      [api.apiMetadata.airnode]: [
+        ...(gatewaysObject?.[api.apiMetadata.airnode] || []),
```

Comment:
piggybacking from the previous comment, in this instance I cannot even reference `[api.apiMetadata.airnode]` unless I specify the type as Gateways.